libraries/Tlc5490_SAMD/tlc_config.h

```diff
@@ -61,7 +61,7 @@
     of the first TLC to the SIN (TLC pin 26) of the next.  The rest of the pins
     are attached normally.
     \note Each TLC needs it's own IREF resistor */
-#define NUM_TLCS    3
```

Comment:
This might mess up the daisy-chained modules we have. @brandogw do we want this default installed like this? We could also update the arduino code to have this set at a higher level so that libs don't need to be changed if you upload to a daisy-chained vs. individual.

Reply:
The fluidics module uses 3 TLC chips that are daisy chained. I think the NUM_TLC variable shouldn't be impacted by the edits but should be tested if possible.

Reply:
Yes, NUM_TLC doesn't affect this recent fix. Everything works with NUM_TLC set to 3 as well. I have tested it with our evolver. I just changed it to 1 because we don't have any daisy chained TLCs in our setup.

Reply:
Awesome, I'll give it a go before merging as well.

Reply:
Tested - works for fluidic module in daisy chained setup. Going to go ahead and merge.